potassium/potassium.py

```diff
@@ -83,7 +83,10 @@ def _handle_generic(self, route, endpoint, flask_request):
                     return res
                 with self._lock:
                     response = endpoint.func(req).json
-                self.event_chan.put(item = True, block=False)
+                try:
```

Comment:
looks like an unrelated bugfix made it into this PR. Noting it here, but will still merge.